bigbluebutton-html5/imports/ui/components/settings/submenus/notification/component.jsx

```diff
@@ -186,6 +192,30 @@ class NotificationMenu extends BaseMenu {
               </Styled.FormElementCenter>
             </Styled.Col>
           </Styled.Row>
+          {isLiveKit ? (
+            <Styled.Row>
+              <Styled.Col>
+                <Styled.Label aria-hidden>
+                  {intl.formatMessage(intlMessages.muteUnmuteLabel)}
+                </Styled.Label>
+              </Styled.Col>
+              <Styled.Col>
+                <Styled.FormElementCenter>
+                  {displaySettingsStatus(settings.muteUnmuteAudioAlerts)}
+                  <Toggle
+                    icons={false}
+                    defaultChecked={settings.muteUnmuteAudioAlerts}
+                    onChange={() => this.handleToggle('muteUnmuteAudioAlerts')}
+                    ariaLabel={`${intl.formatMessage(intlMessages.muteUnmuteLabel)} ${intl.formatMessage(intlMessages.audioAlertLabel)} - ${displaySettingsStatus(settings.muteUnmuteAudioAlerts, true)}`}
+                    showToggleLabel={showToggleLabel}
+                  />
+                </Styled.FormElementCenter>
+              </Styled.Col>
+              <Styled.Col>
+                {/* Empty column to align with other rows */}
```

Comment:
See internal inquiry made to the design team about the "empty" spot for the popup alert. Follow their orientation once they reply.

<img width="679" height="476" alt="Image" src="https://github.com/user-attachments/assets/07cacd65-20ce-4e01-b2ee-f12463526c57" />